inst/csv/replacementPatterns.csv

```diff
@@ -247,4 +247,50 @@ From,To,Pattern,Replacement
 "sql server","impala",".dbo.","."
 "sql server","impala","##","<hash><hash>"
 "sql server","impala","#","%temp_prefix%%session_id%"
-"sql server","impala","<hash><hash>","##"
\ No newline at end of file
+"sql server","impala","<hash><hash>","##"

```

Comment:
Ok, thanks @schuemie, also looks like I put this comment on an 'impala' translation, but this is also seen in the 'netezza' list of translations too.  @tomwhite might want to review his transltation patterns he implemented for impala and temp tables, but I'll open that as a separate issue related to his PR.